battle_of_the_neighbourhoods.ipynb

```diff
@@ -125,7 +125,7 @@
   },
```

Comment:
<p>Nice work catching that error :) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/anneadb/Coursera_Capstone/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>